backend/routes/stock.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const router = express.Router();
+const stockService = require('../services/stock');
+
+router.get('/',async (req,res)=>{
+    try{
+        const response = await stockService.getAll();
+        res.status(200).json(response);
+
+
+    }catch (e) {
+        res.status(503)
```

Comment:
Will the error handling be addressed in another PR ?

Reply:
I have tried to implement the endpoints quickly that's why I could not handle the errors properly.
Next, it will be handled